### ISSUE TITLE
implement a simplified UX for starting a company

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3529]
Reticketed from @webmaven at https://github.com/gratipay/inside.gratipay.com/issues/180#issuecomment-100501970:

> In a sense, the problem you're describing is already solved for open source by [Fiscal Sponsors](https://lwn.net/Articles/548542/) like the [Software Freedom Conservancy](https://sfconservancy.org/), at least for projects that are too small to merit their own 501(c)3 (eg. not [The jQuery Foundation](https://jquery.org/team/)). I do not know how small their smallest project is. [...]
> 
> Can Gratipay do anything to help individuals (or small teams) either find an appropriate Fiscal Sponsor or establish a corporation to get them over that hump?

I'm thinking of how Stripe abstracted away merchant accounts for people, but really more like Braintree, because we want to allow people to bring their own corporation (Braintree allows you to bring your own merchant account and just use them as a gateway, _or_ use their simplified one-size-fits-all merchant account).

We're wrestling with Gratipay's structure on https://github.com/gratipay/inside.gratipay.com/issues/242 wrt @rohitpaulk's relationship to Gratipay, under pressure to bring back payroll (#3433).
##

**Comment by whit537** [comment]
Closing in favor of #2569.

**Comment by whit537** [comment]
Actually, reopening, and making #2569 more general. We need to understand the underlying problem we're trying to solve, and decide if a simplified UX for solving a company is the right approach. There may be other possible approaches.

**Comment by whit537** [comment]
Actually, instead of making #2569 more general, let's use https://github.com/gratipay/inside.gratipay.com/issues/242 for the top-level discussion.

**Comment by whit537** [comment]
Blocked on https://github.com/gratipay/inside.gratipay.com/issues/242.

**Comment by webmaven** [comment]
This might be a point at which you should ask bkuhn about what the bookkeeping arrangements are for Fiscal Sponsors, and what the arrangement between a Fiscal Sponsor and the Gratipay *project* vs. the *LLC* would be like.

**Comment by whit537** [comment]
@webmaven Yup. Let's include fiscal sponsorship as an option on https://github.com/gratipay/inside.gratipay.com/issues/242.

**Comment by techtonik** [comment]
"fiscal sponsorship" - https://github.com/gratipay/inside.gratipay.com/issues/243
